YakimS/assignment1/Q2

```diff
@@ -0,0 +1,89 @@
+public class Assignment1Q2 {
```

Comment:
Like in Q1, could you separate "business" part of code (`MyLinkedList` class) and the testing one (`main` method)?